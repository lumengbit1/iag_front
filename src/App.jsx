import React from 'react';
import PropTypes from 'prop-types';

const App = (props) => (
  <div className="App">
    {props.children}
  </div>
);

App.propTypes = {
  children: PropTypes.node.isRequired,
};

export default App;
