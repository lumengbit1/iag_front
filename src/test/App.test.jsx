import React from 'react';
import { render } from '@testing-library/react';
import App from '../App';

describe('Test <App />', () => {
  it('case: expect rendering correct', () => {
    const { container } = render(
      <App>
        <span />
      </App>,
    );
    expect(container).toMatchSnapshot();
  });
});
