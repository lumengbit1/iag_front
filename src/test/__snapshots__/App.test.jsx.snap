// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <App /> case: expect rendering correct 1`] = `
<div>
  <div
    class="App"
  >
    <span />
  </div>
</div>
`;
